html/cross-origin-embedder-policy/reporting-navigation.https.html

```diff
@@ -110,40 +110,38 @@
     function desc(s) {
       return s === '' ? '(none)' : s;
     }
-    async_test(async (t) => {
-      try {
-        const targetUrl = REMOTE_FRAME_URL + testcase.target;
-        const parent =
-          await loadFrames(t, testcase.parent, withEmptyFrame, targetUrl);
-
+    // These tests are very slow, so they must be run in parallel using
```

Comment:
https://wpt.fyi/results/html/cross-origin-embedder-policy/reporting-navigation.https.html?diff&filter=ADC&run_id=646740002&run_id=652370001 shows a new harness error for this test in Safari. I can't see the `new global.ReportingObserver` call but I guess it's here somewhere and not correctly wrapped?

Reply:
Good catch, because I forgot to `catch`. This is why `async_test` + promises is painful :(.